gulpfile.js

```diff
@@ -437,6 +437,10 @@
 			logToTerminal.apply(this, arguments);
 		},
 
+		envOrArg: (...args) => {
```

Comment:
Is it better to do this or just move the implementation of envOrArg here?

Reply:
@erik-ciq please confirm with andy that he is satisfied and resolve this conversation.